src/routes/docs/products/databases/backups/+page.markdoc

```diff
@@ -0,0 +1,58 @@
+---
+layout: article
+title: Database Backups
+description: Learn how to efficiently back up your databases on Appwrite Cloud, ensuring data security and seamless recovery.
+---
+
+Appwrite Backups enables seamless, **encrypted** database backups on Cloud.
+Learn how to efficiently back up your databases, ensuring data security and smooth recovery.
+
+{% info title="Backups are an Appwrite Cloud feature and available on the Enterprise plan" %}
+{% /info %}
+
+Appwrite Backups enable you to automate database backups using Backup Policies, with support for both pre-defined and custom retention options. You can also create manual backups whenever necessary.
+
+# Backups {% #backups %}
+
+## Create Backups {% #create-backups %}
```

Comment:
This is a section about policies. I would adjust the title, start by explaining what a policy is, and then have another sub-heading with the steps to create a policy. 

The same way we might want to add a section for manual backups.